simplified-app-ekirjasto/src/main/assets/txnative/en/txstrings.json

```diff
@@ -1404,6 +1716,9 @@
       "settingsPasswordShow" : {
          "string" : "Show password"
       },
+      "settingsPlaceholder" : {
+         "string" : "Lorem ipsum dolor sit amet, probatus volutpat has at, vis adhuc iuvaret omittantur an, sadipscing conclusionemque his ad. Eam ut simul tempor. Primis consetetur appellantur vim eu, eum an iudico dolorum. Ad vis utamur honestatis, sanctus debitis referrentur an eam."
```

Comment:
This should be removed from all languages in Transifex, the original Android string has been removed, but it still exists in Transifex.